src/WebJobs.Script.WebHost/Middleware/HostWarmupMiddleware.cs

```diff
@@ -50,7 +50,7 @@ private void PreJitPrepare(string jitTraceFileName)
         {
             // This is to PreJIT all methods captured in coldstart.jittrace file to improve cold start time
             var path = Path.Combine(
-                Path.GetDirectoryName(new Uri(typeof(HostWarmupMiddleware).Assembly.CodeBase).LocalPath),
+                Path.GetDirectoryName(new Uri(typeof(HostWarmupMiddleware).Assembly.Location).LocalPath),
```

Comment:
We do need to flag this. Some of the paths are not safe for this property.

Reply:
https://github.com/Azure/azure-functions-host/pull/7323 is already filed.